app/helpers/catalog_helper.rb

```diff
@@ -0,0 +1,56 @@
+module CatalogHelper
+  include Blacklight::CatalogHelperBehavior
+
+  def database_electronic_access?
+    @document['format'].include?('Database') && @document['url_tesim'].present?
+  end
+
+  # These are typical electronic access links and appear at the top of the page
+  # for example
+  # (Database) catalog/11709037
+  # (University of Alberta Access) catalog/2566934
+  # (Finding Aid) catalog/4072071
+  def electronic_access_top?
```

Comment:
```suggestion
  def electronic_access_top?
    database_electronic_access? || @ua_urls.present? || supplimentary_urls.present?
  end
```

This would do the exact same thing as your code.

Another way to think of it is: 

```
 if database_electronic_access? || @ua_urls.present? || supplimentary_urls.present?
    true
  else
    false
  end
```